mergebom.py

```diff
@@ -24,24 +24,29 @@
 import argparse
 import ConfigParser
 import re
-from lib import cfg
+from lib import cfg,lib,report
 from mergebom_class import *
 from datetime import datetime
    
 if __name__ == "__main__":
+    rep = report.Report(log_on_file=True, terminal=True, report_date=None)
 
     file_list = []
     parser = argparse.ArgumentParser()
     parser.add_argument('--workspace-file', '-w', dest='ws', 
                         help='Dove si trova il file WorkSpace', default=None)
+    parser.add_argument('--nome-file.xlsx', '-nw', dest='nw', 
+                        help='Nome file da mergiare', default='bom-')
+    parser.add_argument('--caratteristiche-file', '-cf', dest='cf', action="store_true",
```

Comment:
Il nome non è molto evocativo.. non si capisce molto che cosa fa..
Anche le variabili, dagli un nome più sensato e lungo, non sei in javascript che il nome della variabile deve essere corto..